### DISCUSSION TITLE
Github Support İgnores me for 57 days!!

### DISCUSSION THREAD

**Discussion by mferitsimsek** [community/community#133603] [Other Feature Feedback, Questions, & Ideas] [answered]
I created a ticket from github support and I received an automatic e-mail saying "We have received your request".  

Even though I replied to that e-mail 7 times, I did not receive any response.
 
No matter how busy Github Support is, 57 days is a disgraceful response time for a large company like Github.

Not: I didn't write my ticket number on purpose because they give me an automatic response and close the discussion.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @mferitsimsek, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
